### DISCUSSION TITLE
Only incremental backups? Backups Retention Policy? (if using VolumeSnapshots)

### DISCUSSION THREAD

**Discussion by supermario18b** [cloudnative-pg/cloudnative-pg#3465] [Q&A] [answered]
Hi everyone,

I didn't understand well, how backups and restores work (if using VolumeSnapshots).
Do "ScheduledBackup" and "Backup" Resources only do incremental backups? If so all backups are needed for a Restore.
If it is not like that, is it possible to set a Retention Policy for the backups?

Thanks in advance for some clarification,

supermario18b

**Comment by gbartolini** [comment, marked as answer]
When it comes to incremental/differential, it is all transparent to you.

We currently do not have any retention policy algorithm in place, but it is much much easier now that objects are directly in Kubernetes and you can use labels to prune old data.

**Comment by supermario18b** [comment]
Thank you for the reply @gbartolini
